lib/src/rtc_engine.dart

```diff
@@ -1025,8 +1025,8 @@ class RtcEngine with RtcEngineInterface {
 
   @override
   Future<void> setCloudProxy(CloudProxyType proxyType) {
-    return _invokeMethod('enableRemoteSuperResolution',
-        {'proxyType': CloudProxyTypeConverter(proxyType).e});
+    return _invokeMethod('setCloudProxy',
+        {'proxyType': CloudProxyTypeConverter(proxyType).e.index});
```

Comment:
you should remove `.index`.

Reply:
I believe .index have to be added or you’ll get same error